Node/core/src/bots/ChatConnector.ts

```diff
@@ -103,6 +107,7 @@ export class ChatConnector implements IConnector, IBotStorage {
     private botConnectorOpenIdMetadata: OpenIdMetadata;
     private emulatorOpenIdMetadata: OpenIdMetadata;
     private refreshingToken: Promise.IThenable<string>;
+    private credentialsCache: any;
```

Comment:
nit: we should avoid `any`

Reply:
fixed by creating 'ICredentialsCache' interface